controls/V-72049.rb

```diff
@@ -69,7 +69,6 @@
 user must log on to their account first and then switch the user to the application
 account with the correct option to gain the account’s environment variables."
 
-  # @todo - test for values more restrictive than 077
   file_lines = command('grep -i -s umask /home/*/.*').stdout.split("\n")
```

Comment:
This is going to explode the heck out of NFS and also miss home directories.

You should query the current user list using `getent passwd` and go from there probably.

To test this properly you should *probably* drop to a shell using `runuser` and actually run the `umask` command. There are a LOT of ways for this to go wrong and, in our case, we set everything globally in `/etc/profile.d` so this check wouldn't pick anything up.